mifospay/src/main/java/org/mifos/mobilewallet/mifospay/data/firebase/api/services/MifosPayInstanceIDService.java

```diff
@@ -44,17 +44,19 @@
     // [START refresh_token]
     @Override
     public void onTokenRefresh() {
-        // Get updated InstanceID token.
-        String refreshedToken = FirebaseInstanceId.getInstance().getToken();
-        Log.d(TAG, "Refreshed token: " + refreshedToken);
-
-        // Saving reg id to shared preferences
-        mPreferencesHelper.setFirebaseRegId(refreshedToken);
-
-        // If you want to send messages to this application instance or
-        // manage this apps subscriptions on the server side, send the
-        // Instance ID token to your app server.
-        sendRegistrationToServer(refreshedToken);
+        if (mPreferencesHelper != null) {
```

Comment:
Remove this check and inject `mPreferencesHelper` properly

Reply:
Done